src/App.test.js

```diff
@@ -1,9 +0,0 @@
-import React from 'react';
```

Comment:
Deleting this can effect the tests run by TravisCI. It's probably better to keep this and edit the travis.yml file so it runs the test on the client directory for now.

Reply:
See FB chat @Mera-Gangapersaud

Reply:
Something like this: https://github.com/codecov/example-node/blob/a8a5074313fece463ddf5e6bb2658af01dae374a/.travis.yml#L11